Makefile

```diff
@@ -208,11 +200,50 @@ ifeq ($(quickstart), true)
 	./scripts/package_quickstart.sh deploy/$(target)/manifests/$(ver) deploy/$(target)/quickstart/olm.yaml deploy/$(target)/quickstart/crds.yaml deploy/$(target)/quickstart/install.sh
 endif
 
-##########################
-#  OLM - Commands        #
-##########################
+################################
+#  OLM - Install/Uninstall/Run #
+################################
 
 .PHONY: run-console-local
 run-console-local:
 	@echo Running script to run the OLM console locally:
 	. ./scripts/run_console_local.sh
+
+.PHONY: uninstall
+uninstall:
+	@echo Uninstalling OLM:
+	- kubectl delete crd deploy/upstream/quickstart/crds.yaml
+	- kubectl delete crd deploy/upstream/quickstart/olm.yam
+	- kubectl delete crd catalogsources.operators.coreos.com
+	- kubectl delete crd clusterserviceversions.operators.coreos.com
+	- kubectl delete crd installplans.operators.coreos.com
+	- kubectl delete crd operatorgroups.operators.coreos.com subscriptions.operators.coreos.com
+	- kubectl delete ns olm
+	- kubectl delete ns openshift-operator-lifecycle-manager
+	- kubectl delete ns openshift-operators
+	- kubectl delete ns operators
+	- kubectl delete clusterrole.rbac.authorization.k8s.io/aggregate-olm-edit
+	- kubectl delete clusterrole.rbac.authorization.k8s.io/aggregate-olm-view
+
+
+.PHONY: run-local
+run-local: build-linux
+	@echo Building OLM:
+	rm -rf build
+	. ./scripts/build_local.sh
+	mkdir -p build/resources
+	. ./scripts/package_release.sh 1.0.0 build/resources Documentation/install/local-values.yaml
+	. ./scripts/install_local.sh local build/resources
+	rm -rf build
+
+.PHONY: install-crd
+install-crd:
+	@echo Applying upstream quick start crd
+	- kubectl apply -f deploy/upstream/quickstart/crds.yaml
+	- kubectl apply -f deploy/upstream/quickstart/olm.yaml
+
+.PHONY: install-ocp
```

Comment:
If we check the script to install it can be there. However, if the yaml were not applied has no problem at all since the - will ignore any error faced.